### ISSUE TITLE
Spyder is not starting anymore

### ISSUE THREAD

**Issue by Lugle29** [spyder-ide/spyder#21217]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
Since yesterday all of a sudden spyder won't launch anymore. If it is via a single distribution, via Anaconda Prompt or Anaconda Navigator just nothing happens. If I try to launch in Anaconda I get the following "Exit code: 3221225477". I tried everything I found so far, like updating, reinstalling, installing spyder in a new environment and even reset my whole windows. I also watched the Spyder-YouTube Video which should troubleshoot but nothing helped.

### What steps reproduce the problem?

1. Clicking on Spyder Icon in Windows
2. Writing "spyder" in Anaconda Prompt
3. Clicking on "launch" in Anaconda Navigator

### What is the expected output? What do you see instead?
Expected ouptut would be a launch of Spyder, I doesn't see anything instead of the exit code in Anaconda-Navigator

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 5.4.3
* Python version: 3.11.3
* Qt version: 5.15.7
* PyQt version: 5.15.2
* Operating System name/version: Windows 11 x64

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
Hey @Lugle29, thanks for reporting. Please open the Anaconda Prompt, run there

    spyder --debug-info verbose --debug-output file

and upload here the file called `spyder-debug.log` that is created in the same directory where you ran the above command.

That will help us to understand what's happening in your case.

**Comment by Lugle29** [comment]
[spyder-debug.log](https://github.com/spyder-ide/spyder/files/12284004/spyder-debug.log)

Hey Carlos,
thank you for replying and at least, spyder opened up, when I use the command, you gave me. The log should be attached to this message. Fortunately I'm able to start Spyder again, but when I run a command I get a warning (see attached picture).

<img width="341" alt="image" src="https://github.com/spyder-ide/spyder/assets/141414517/3a29c36d-3ba6-46b0-9435-57b4f451df43">

**Comment by ccordoba12** [comment]
> thank you very much for replying and at least, spyder opened up, when I use the command, you gave me.

Ok, that's great news!

> The log should be attached to this message.

I looked at it and don't see anything strange. So, probably the issue is with Anaconda Navigator that it's unable to start Spyder. So, please open the Anaconda Prompt and run there simply:

    spyder

That should work for you.

> but when I run a command I get a warning (see attached picture).

We're aware of that problem and it's already fixed in our latest version (5.4.4), which is not part of Anaconda yet. However, you can safely ignore it because it's a harmless warning.

But I'd like to ask you: is it shown after every command you run in the console? Or just the first one?

**Comment by Lugle29** [comment]
Hey Carlos, starting spyder with the command works and even through anaconda navigator I'm able to start spyder again. So thank you very much for your help!

The warning message is just shown after the first command. After this everything works without warnings or problems.

**Comment by ccordoba12** [comment]
Great! I'm glad Spyder is working for you again. Given that and the explanation I gave you about the warning in the console, I'm going to close this issue.

**Comment by Lugle29** [comment]
Hello Carlos, unfortunately I encounter the same problem again and again deinstalling and reinstalling isn't working. Even trying the code you gave me `spyder --debug-info verbose --debug-output file` isn't executing anything and the text file which will be created by this is empty. Do you have any advice?

**Comment by ccordoba12** [comment]
Please open the Anaconda Prompt and then run there:

```python
In [1]: from spyder.app import start
In [2]: start.main()
```

That should show you what errors are preventing Spyder to start.

**Comment by Lugle29** [comment]
Thank you for the fast reply but unfortunately following the last command, nothing happens.

<img width="865" alt="image" src="https://github.com/spyder-ide/spyder/assets/141414517/ce26a787-05a7-4c41-9a15-6ac0bfab2741">

**Comment by ccordoba12** [comment]
So, after running `start.main()` IPython is closed and you return to the normal cmd prompt?

**Comment by Lugle29** [comment]
Yes, it seems like the program tries to start but then nothing happens and IPython closes so that I'm thrown back to the base Anaconda Promt

**Comment by ccordoba12** [comment]
Ok, then run these commands instead in IPython again:

```python
In [1]: from spyder.app import start
In [2]: start.CLI_OPTIONS.debug_info = 'verbose'
In [3]: start.main()
```

That hopefully should print something to the terminal that let us know what's happening in your case.

**Comment by Lugle29** [comment]
Hey Carlos, those commands still don't lead to any printing but I found a solution. I'm working currently with a laptop which is connected to a Lenovo ThinkPad Docking Station to use another screen. When I disconnect the dock, Spyder is starting, when connected I'm running into this issue. However, if I start Spyder and connect the laptop afterwards, Spyder is still working. Probably it's a problem with the firmware of the docking station.

**Comment by ccordoba12** [comment]
Ok, thanks for the update @Lugle29. Then I think this problem is a duplicate of issue #20344.

There you'll find [a workaround](https://github.com/spyder-ide/spyder/issues/20344#issuecomment-1402430390) until we implement a proper fix. Please try it and let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.

I think this is a duplicate of issue #20344, which will be fixed in our next version (5.5.0), to be released next month.
